testGameSearch/GameService.swift

```diff
@@ -19,14 +19,16 @@ public class GameService {
     }
     */
     
-    func getGames(game: String, completion: @escaping (Game?) -> ()) {
+    func getGames(gameName: String, completion: @escaping (Game?) -> ()) {
         
-        let parameters = "fields name, summary, platforms.name; where platforms = (48,6,49); limit 10;"
+        let parameters = "search \"\(gameName)\"; fields name, summary, platforms.name; where platforms = (48,6,49); limit 10;"
         let postData = parameters.data(using: .utf8)
         
-        let url = URL(string: "https://api-v3.igdb.com/games/\(game)")!
-        // Also not sure if it's a good idea to force unwrap the url
-        // Trying to be able to search the API by a specific game name.
+        guard let url = URL(string: "https://api-v3.igdb.com/games/") else {
+            // Force unwrap would be fine if it's just a static string, but guard let is a better habit to get into.
+            print("Invalid URL source https://api-v3.igdb.com/games/")
+            return
+        }
```

Comment:
Force unwrap is safe for a static string, but if you were putting a user-supplied string into the URL it would be safer to use guard-let and handle the failure.